README.md

```diff
@@ -10,6 +10,10 @@ This standard sets a quality level for codebases that meets the needs of public
 
 The standard lives at [standard.publiccode.net](https://standard.publiccode.net/), and for an overview of all content see [`index.md`](index.md).
 
+[![The printed Standard for Public Code lying on a table between two hands](https://img.youtube.com/vi/QWt6vB-cipE/mqdefault.jpg)](https://www.youtube.com/watch?v=QWt6vB-cipE)
+
+The image above is linked to an introduction video of the Standard for Public Code on YouTube.
```

Comment:
This is really hard to use for those using non-gui-mouse based browsers. It also adds the cognitive overhead of having to relate to other elements before being able to act. A simple link with a good description will serve it's purpose better here.

```suggestion
[The video: Standard for Public Code - Creative Commons Global Summit 2020 (4:12) on YouTube](https://www.youtube.com/watch?v=QWt6vB-cipE)
```